backend/src/Controller/ChatRoom/ChatRoomController.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace App\Controller\ChatRoom;
+
+use App\Controller\BaseController;
+use App\Service\ChatRoom\ChatRoomService;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Serializer\SerializerInterface;
+use OpenApi\Annotations as OA;
+use Nelmio\ApiDocBundle\Annotation\Security;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+
+/**
+ * fetch chat room between admin and user.
+ * @Route("v1/chatroom/")
+ */
+class ChatRoomController extends BaseController
+{
+    private ChatRoomService $chatRoomService;
+
+    public function __construct(SerializerInterface $serializer, ChatRoomService $chatRoomService)
+    {
+        parent::__construct($serializer);
+        $this->chatRoomService = $chatRoomService;
+    }
+
+    /**
+     * store and captain :fetch chat room for user
+     * @Route("chatroom", name="getChatRoom", methods={"GET"})
+     * @return JsonResponse
+     *
+     * @OA\Tag(name="Chat Room")
+     *
+     * @OA\Parameter(
+     *      name="token",
+     *      in="header",
+     *      description="token to be passed as a header",
+     *      required=true
+     * )
+     *
+     * @OA\Response(
+     *      response=201,
+     *      description="Returns chat room info",
+     *      @OA\JsonContent(
+     *          @OA\Property(type="string", property="status_code"),
+     *          @OA\Property(type="string", property="msg"),
+     *          @OA\Property(type="object", property="Data",
```

Comment:
I suppose the API return LIST of chats. According to that, the Data field should be array, right?